x-pack/plugins/ml/server/models/data_recognizer/modules/apache_ecs/ml/datafeed_low_request_rate_ecs.json

```diff
@@ -14,7 +14,7 @@
       "buckets": {
         "date_histogram": {
           "field": "@timestamp",
-          "interval": 900000,
+          "fixed_interval": 900000,
```

Comment:
`"15m"` would be nicer.

Same in the other 3 modules.

Reply:
fixed in 105d2ea65be9531ad1d2739680f4cbd1a3e55b37